core/tests/protractor_desktop/publicationAndLibrary.js

```diff
@@ -217,14 +217,23 @@ describe('Permissions for private explorations', function() {
     explorationEditorSettingsTab = explorationEditorPage.getSettingsTab();
     explorationPlayerPage = new ExplorationPlayerPage.ExplorationPlayerPage();
   });
-  it('should be correct for collaborators', function() {
+  it('should not allow adding users if title is not given', function() {
     users.createUser('alice@privileges.com', 'alicePrivileges');
     users.createUser('bob@privileges.com', 'bobPrivileges');
     users.createUser('eve@privileges.com', 'evePrivileges');
 
     users.login('alice@privileges.com');
     workflow.createExploration();
     explorationEditorPage.navigateToSettingsTab();
+
+    workflow.openEditTestRolesForm();
+    expect(workflow.canAddUser()).toBe(false);
+    workflow.setTitleForExploration('Chuck Norris');
```

Comment:
can we use the existing `ExplorationEditorSettingsTab.setTitle` instead of creating a duplicate function for the same task?

Reply:
I feel its better to use this, here I check if a title exists and then add a title if a title does not exist. So this is a little different in functionality.